templates/menu.ftl

```diff
@@ -49,7 +49,8 @@
                         <li><a href="http://training.nextflow.io">Nextflow training</a></li>
                         <li><a href="http://nextflow-io.github.io/patterns/index.html">Implementation patterns</a></li>
                         <li><a href="https://github.com/nextflow-io/nextflow">GitHub repository</a></li>
-                        <li><a href="https://github.com/nextflow-io/nextflow/discussions">Discussion forum</a></li>
+                        <li><a href="https://community.seqera.io">Discourse forum</a></li>
+                        <li><a href="https://github.com/nextflow-io/nextflow/discussions">GitHub discussions</a></li>
                         <li><a href="https://www.nextflow.io/slack-invite.html">Slack community chat</a></li>
```

Comment:
Up for discussion: should we remove these two links right away?

My preference is yes - remove.
```suggestion
```

Reply:
Yeah, I would too, agree to remove them.

Reply:
Not for now.